geode-redis/src/distributedTest/java/org/apache/geode/redis/internal/executor/InfoDUnitTest.java

```diff
@@ -116,7 +114,7 @@ public void testConcurrentInfoFromDifferentServers_doNotStepOnEachOther() {
           int commandsProcessed2 = Integer.valueOf(info2.get(COMMANDS_PROCESSED));
           assertThat(commandsProcessed2).isGreaterThanOrEqualTo(previousCommandsProcessed2.get());
           previousCommandsProcessed2.set(commandsProcessed2);
-        }).runInLockstep();
+        }).run();
```

Comment:
Just wanted to make sure this was intentionally changed.

Reply:
yes, it was, thanks for checking